students/chieu_quach/lesson08/circle.py

```diff
@@ -0,0 +1,159 @@
+
+import math
+
+class Circle(object):
+ 
+    def __init__(self, radius):
+
+        self.radius = radius
+
+    @property
+    def get_diameter(self):
+   # def get_diameter(self, value):
+      
+        return self.radius * 2   
+
+    @get_diameter.setter
+    def get_diameter(self, value):
+        self.radius = value
+
+   
+    def area(self):
+        
+        PI = 3.14
+        return PI * self.radius * self.radius    
+           
+    def perimeter(self):
+ 
+        return 2*math.pi*self.radius
+
+          
+    # Add two Cicle values
+    def __add__(self, other):
+
+        total_num = self.radius + other.radius
+        return Circle(total_num)
+
+    def __radd__(self):
+        
+        return self.__add__(other)
+
+    # Multiply two Cicle values (use __mul__  and __rmul)
+    def __mul__(self, other):
+
+        total_num = self.radius * 3
+        return Circle(total_num)
+
+    def __rmul__(self):
+        
+        return self.__mul__(other)
+
+    def __repr__(self):
+        return repr(self.radius)
+
+    def __str__(self):
+        return str(self.radius)
```

Comment:
the idea with `__str__` and `__repr__` is that you customize what gets printed out